presto-common/src/main/java/com/facebook/presto/common/RuntimeMetricName.java

```diff
@@ -55,4 +55,5 @@ private RuntimeMetricName()
     public static final String STORAGE_READ_TIME_NANOS = "storageReadTimeNanos";
     // Size of the data retrieved by read call to storage
     public static final String STORAGE_READ_DATA_BYTES = "storageReadDataBytes";
+    public static final String TABLE_WRITER_NUM_WRITTEN_FILES = "numWrittenFiles";
```

Comment:
"numWrittenFiles" -> "writtenFilesCount"
so that its similar to other metric names in this file.